pkg/cloudprovider/.import-restrictions

```diff
@@ -7,7 +7,8 @@
 				"k8s.io/cloud-provider",
 				"k8s.io/component-base",
 				"k8s.io/klog",
-				"k8s.io/utils"
+				"k8s.io/utils",
+                                "k8s.io/mount-utils"
```

Comment:
intent seems can be updated

Reply:
seems indent is correct when u open raw file